nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -116,11 +126,13 @@ protected void initialize(Map<String, String> properties) {
         }
         region = properties.get(BUCKET_REGION_PROPERTY);
         bucket = properties.get(BUCKET_NAME_PROPERTY);
-        bucketPrefix = defaultIfEmpty(properties.get(BUCKET_PREFIX_PROPERTY), "");
-        roleArn = properties.get(ROLE_ARN_PROPERTY);
+        bucketPrefix = defaultIfEmpty(properties.get(BUCKET_PREFIX_PROPERTY), "/");
```

Comment:
Why this change? The default should be an empty string. This is a prefix we add to every key, by default they should be unchanged.